### PR TITLE
resources and color alignment

### PR DESCRIPTION
**Pull request by rzulian**
I've aligned colors in playersInfo.
Complete resource descriptions
I've also added the lostPlanet in the researchTrack. Even though it's not that good.

### CODE REVIEW

File: src/components/ResearchTile.vue [hunk 1]
```diff
@@ -33,13 +35,19 @@ import Resource from './Resource.vue';
     federation(): Federation {
       if (this.level == 5 && this.field === ResearchField.Terraforming) {
         return this.$store.state.gaiaViewer.data.terraformingFederation;
+      }    
+    },
+    lostPlanet(): PlanetEnum {
+      if (this.level == 5 && this.field === ResearchField.Navigation) {
```
Comment (coyotte508) [inline src/components/ResearchTile.vue]: This should check that no one actually reached the end of the track. The lost planet should disappear if someone else is on the research tile

File: src/components/Resource.vue [hunk 2]
```diff
@@ -32,8 +32,17 @@ export default class Resource extends Vue {
   level: number;
 
   label(kind,count,level): string {
+       
+    if (kind == 'd') {
+      return this.level == 2 ? "cost 2" : this.level == 3 ? "cost 1" : "";
```
Comment (coyotte508) [inline src/components/Resource.vue]: Maybe add `cost 3` for level 0?

File: src/components/Resource.vue [hunk 3]
```diff
@@ -32,8 +32,17 @@ export default class Resource extends Vue {
...
+    };
+    if (kind == 'r') {
+      return this.level == 2 ? "nav 2" : this.level == 4 ? "nav 3" : this.level == 5 ? "nav 4" : "";
```
Comment (coyotte508) [inline src/components/Resource.vue]: Maybe add `nav 1` for level 0?

File: src/components/ResearchTile.vue (hunk 1 above)
Comment (rzulian) [reply src/components/ResearchTile.vue]: done

**Comment by rzulian** [comment]
@coyotte508 how about this?
screen shot 2018-09-19 at 10 30 59

**Pull request merged**